pkgs/servers/etcd/3.5.nix

```diff
@@ -0,0 +1,76 @@
+{ lib, buildGoModule, fetchFromGitHub, symlinkJoin }:
+
+let
+  etcdVersion = "3.5.1";
+  etcdSrc = fetchFromGitHub {
+    owner = "etcd-io";
+    repo = "etcd";
+    rev = "v${etcdVersion}";
+    sha256 = "sha256-Ip7JAWbZBZcc8MXd+Sw05QmTs448fQXpQ5XXo6RW+Gs=";
+  };
+
+  commonMeta = with lib; {
+    description = "Distributed reliable key-value store for the most critical data of a distributed system";
+    license = licenses.asl20;
+    homepage = "https://etcd.io/";
+    maintainers = with maintainers; [ offline zowoq endocrimes ];
+    platforms = platforms.unix;
```

Comment:
No reason this shouldn't default to `buildGoModule` platform like we usually do. I'll open a PR to remove this from the other `etcd` derivations.

```suggestion
```

Reply:
Cool 👍 - figured there was a historic nix reason, glad there isn't 😅

Reply:
Seems `platforms` from `buildGoModule` isn't working with `symlinkJoin` for some reason so we will need to set it.

`platforms = platforms.darwin ++ platforms.linux;`

Reply:
weird, bumped.